src/extension.js

```diff
@@ -184,38 +184,36 @@ module.exports = {
  */
 async function showMenu()
 {
-	let menuGlobal = [
-		{label: "Connect to host"     , id: 0},
-		{label: "Disconnect from host", id: 1}
+	let dynamicExtMenu = [
+		{label: "Connect to host"     , foo: Menu_sshConnect},
+		{label: "Disconnect from host", foo: Menu_sshDisconnect}
 	];
+
 	let conf_ctagssh = vscode.workspace.getConfiguration('ctagssh');
 	
 	if (conf_ctagssh.usingSSHFS == true) {
-		menuGlobal.push({label: "Set profile on SSH FS ->", id: 2});
+		dynamicExtMenu.push({label: "Set profile on SSH FS ->", foo: Menu_slectSSHfsProfile});
 	}
+	dynamicExtMenu.push({label: "Load CTags file using remote SSH connection", foo: loadRemoteCTags});
```

Comment:
Add this element only then we have right path to remote folder contain ctag files.